fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2179,17 +2179,17 @@ private List<Long> chosenBackendIdBySeq(int replicationNum, Multimap<String, Str
         SystemInfoService systemInfoService = GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo();
         List<Long> chosenBackendIds = systemInfoService.getNodeSelector()
                 .seqChooseBackendIds(replicationNum, true, true, locReq);
-        if (!CollectionUtils.isEmpty(chosenBackendIds)) {
-            return chosenBackendIds;
-        } else if (replicationNum > 1) {
-            List<Long> backendIds = systemInfoService.getBackendIds(true);
+        if (CollectionUtils.isEmpty(chosenBackendIds)) {
+            List<Long> backendIds = systemInfoService.getAvailableBackends().stream()
+                    .filter(b -> !b.checkDiskExceedLimitForCreate()).map(Backend::getId).collect(Collectors.toList());
             throw new DdlException(
-                    String.format("Table replication num should be less than or equal to the number of available BE nodes. "
-                            + "You can change this default by setting the replication_num table properties. "
-                            + "Current alive backend is [%s]. ", Joiner.on(",").join(backendIds)));
-        } else {
-            throw new DdlException("No alive nodes");
+                    String.format("Table replication num should be less than or equal to the number of available backends. "
```

Comment:
also add the number of backends that have not enough disk space in the message?

Reply:
done